serializers/reviewable_akismet_user_serializer.rb

```diff
@@ -7,6 +7,10 @@ class ReviewableAkismetUserSerializer < ReviewableSerializer
 
   attributes :user_deleted
 
+  def created_from_flag?
+    true
+  end
```

Comment:
The only use for this is to determine whether we display the context question:

> Is this X Y?

I think it's not a stretch to say that Akismet is flagging posts. We gave the same treatment to Akismet flagged posts.